src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -2186,6 +2188,1317 @@ struct areatrigger_find_the_lost_expedition_follower : AreaTriggerAI
+    SPELL_KNOCKBACK                                   = 320735,
+    SPELL_CHARGE_KNOCKBACK_DRUID                      = 320767,
+    SPELL_CHARGE                                      = 100,
+    SPELL_SLAM                                        = 1464,
+    SPELL_CHARGE_KNOCKBACK_WARRIOR                    = 320583,
+    SPELL_SHIELD_OF_THE_RIGHTEOUS                     = 53600,
+    SPELL_CRUSADER_STRIKE                             = 35395,
+    SPELL_SINISTER_STRIKE                             = 1752,
+    SPELL_EVISCERATE                                  = 196819,
+    SPELL_SMITE                                       = 585,
+    SPELL_SHADOW_WORD_PAIN                            = 589,
+    SPELL_CHARGE_KNOCKBACK                            = 320605,
+    SPELL_PRIMAL_STRIKE                               = 73899,
+    SPELL_LIGHTNING_BOLT                              = 188196,
+    SPELL_FIRE_BLAST                                  = 319836,
+    SPELL_FROSTBOLT                                   = 116,
+    SPELL_CORRUPTION                                  = 172,
+    SPELL_SHADOWBOLT                                  = 686,
+    SPELL_MOONFIRE                                    = 164812,
+    SPELL_WRATH                                       = 5176,
+    SPELL_TIGER_PALM                                  = 100780,
+    SPELL_BLACKOUT_KICK                               = 100784,
+    SPELL_RANGED_ROOT_DNT                             = 320608,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_WARRIOR              = 320741,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_PRIEST               = 320649,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_SHAMAN               = 320705,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_WARRIOR_MAGE         = 320741,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_WARLOCK              = 320606,
+    SPELL_AGGRO_RADIUS_CHECK_DNT_DRUID                = 320766
+};
+
+struct npc_sparring_partner_combat_training : public ScriptedAI
+{
+    npc_sparring_partner_combat_training(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        _events.ScheduleEvent(EVENT_COMBAT_TRAINING_WALK_AND_TALK, 2s);
+    }
+
+    void IsSummonedBy(WorldObject* summonerWO) override
+    {
+        Unit* summoner = summonerWO->ToUnit();
+
+        if (!summoner || !summoner->IsPlayer())
+            return;
+
+        Player* player = summoner->ToPlayer();
+
+        _playerGUID = player->GetGUID();
+
+        _playerClass = player->GetClass();
+
+        if (player->GetTeam() == ALLIANCE)
+        {
+            _allianceGUID = me->GetGUID();
+            _hordeGUID = ObjectGuid::Empty;
+            _summonSpellAura = SPELL_SUMMON_CAPTAIN_GARRICK_COMBAT;
+
+            if (_playerClass == CLASS_MONK)
+                _questID = QUEST_ENHANCED_COMBAT_TACTICS_ALLIANCE_MONK;
+            else
+                _questID = QUEST_ENHANCED_COMBAT_TACTICS_ALLIANCE;
+        }
+        else
+        {
+            _allianceGUID = ObjectGuid::Empty;
```

Comment:
can be removed. ObjectGuids are default initialized as empty and since this is the very first action that triggers in this script, this operation is not needed. same to _hordeGuid  above this one here

Reply:
Will test